articles/cognitive-services/text-analytics/includes/quickstarts/python-sdk.md

```diff
@@ -172,8 +172,7 @@ def authenticate_client():
     ta_credential = AzureKeyCredential(key)
     text_analytics_client = TextAnalyticsClient(
             endpoint=endpoint, 
-            credential=ta_credential, 
-            api_version=TextAnalyticsApiVersion.V3_0)
+            credential=ta_credential)
```

Comment:
Hey @suhas92 in the email I actually suggested still including the code for `api_version`, just having it commented out, since we assume while 5.0.0 is the latest (only) GA package, most users of service's 3.0 will be using that package.

Can you add a comment on this line saying something like "use this code if your SDK version is 5.0.0"

Then can you insert before line 176 something like
```python
# uncomment the following code to initialize a client if your SDK version is a 5.1.0 beta version, 
# and are looking to specify a service version of 3.0 for your client

# from azure.ai.textanalytics import TextAnalyticsApiVersion
# text_analytics_client = TextAnalyticsClient(
#   endpoint=endpoint,
#   credential=ta_credential,
#   api_version=TextAnalyticsApiVersion.V3_0
# )
```

Reply:
Updated as suggested, p[lease take a look. @iscai-msft